channels/utils.py

```diff
@@ -105,6 +105,8 @@ def translate_praw_exceptions():
             raise NotFound() from exc
         elif exc.error_type == 'SUBREDDIT_EXISTS':
             raise ConflictException() from exc
+        elif exc.error_type == 'DELETED_COMMENT':
+            raise GoneException() from exc
```

Comment:
do you think it's worthwhile to test this? It seems like we are exercising this code via the view test you wrote, not sure if there's value in explicitly testing this `translate_praw_exceptions` function by itself.

Reply:
We don't test this anywhere else either, maybe address as a separate PR so we can cover the whole func?

Reply:
I'm not super worried about it, just wanted to raise it if we thought there was a lot of value in having unit tests for it. Maybe a tech debt issue would be the way forward there.

Reply:
Add #487